app/main.py

```diff
@@ -1 +1,14 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_name, source_file, destination_file, *_ = command.split()
+    if command_name != "cp":
+        raise ValueError(
+            "Invalid command format."
+            "Expected format: cp source_file destination_file"
+        )
+    if source_file == destination_file:
+        raise ValueError(
+            "The source file and the destination file have the same names."
+        )
+    with open(source_file, "r") as file_in:
+        with open(destination_file, "w") as file_out:
```

Comment:
In case python is a limiting factor, update it to the python 3.10 to place such conditions inside of `()` instead of nesting context managers. Please, resolve an issue mentioned earlier